packages/block-library/src/gallery/gallery.native.js

```diff
@@ -127,7 +129,14 @@ export const Gallery = ( props ) => {
 					);
 				} ) }
 			</Tiles>
-			{ mediaPlaceholder }
+			<View
+				style={ [
+					align === WIDE_ALIGNMENTS.alignments.full &&
```

Comment:
Could you please create a `const isFullWidth = align === WIDE_ALIGNMENTS.alignments.full`

Reply:
Done https://github.com/WordPress/gutenberg/pull/25184/commits/6b75b0ea25a501eb2d78a775a399e04c339218a0 thanks!